requirements-dev.txt

```diff
@@ -9,8 +9,8 @@ watchdog==0.8.3
 freezegun==0.3.4
 
 coverage==3.7.1
-flake8==3.4.1
+flake8==2.6.2
```

Comment:
Why are we going for an old version?  Shiny and new is good?

Reply:
Updated PR desc for this